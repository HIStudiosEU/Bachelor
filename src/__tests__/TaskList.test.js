// TaskList.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import TaskList from '../components/TaskList';

describe('TaskList', () => {
  it('rendert korrekt mit leeren Aufgaben', () => {
    const tree = renderer.create(<TaskList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
