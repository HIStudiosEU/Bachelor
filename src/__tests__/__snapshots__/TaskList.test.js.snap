// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskList rendert korrekt mit leeren Aufgaben 1`] = `
<div
  className="taskListContainer"
>
  <h2>
    ToDo-List
  </h2>
  <input
    className="taskInput"
    onChange={[Function]}
    placeholder="Neue Aufgabe hinzufügen"
    type="text"
    value=""
  />
  <button
    className="addTaskButton"
    onClick={[Function]}
  >
    Hinzufügen
  </button>
  <ul
    className="taskList"
  />
</div>
`;
